report_app/main/views.py

```diff
@@ -206,66 +209,11 @@ def unknown_server_error(err):
     return render_template("500.html"), 500
 
 
-@main.route("/public-github-repositories.html")
```

Comment:
Do we need to keep these routes as well? 🤔 `/public-github-repositories.html` and `/private-github-repositories.html`